app/addon_cmds/commands.py

```diff
@@ -1,13 +1,43 @@
 import os
+from multiprocessing import Process
 
 import click
 from constants import TIMEOUT_30MIN
 from ocm_python_wrapper.cluster import ClusterAddOn
 from ocm_python_wrapper.ocm_client import OCMPythonClient
 
 
+def run_action(action, addons, parallel, timeout):
+    jobs = []
+    for values in addons.values():
+        addon_action_func = getattr(values["cluster_addon"], action)
+        _args = [True, timeout]
+        if action == "install_addon":
+            _args.insert(0, values["parameters"])
+
+        if parallel:
+            job = Process(target=addon_action_func, args=tuple(_args))
```

Comment:
Not sure if it should be implemented here or within openshift-ci config, but do you think we should provide a flag for each process that will set to fail the whole flow if an addon installation is failing or passing?

For example while installing RHOAM&RHODS, RHODS has exceeded timeout, and I want the whole flow to continue and not teardown.

Reply:
Update the code, if one of the jobs failed we exit